canonical-websites/www.ubuntu.com
# update download-thank-you-linuxone to vbt

## Done

* updated /download/server/thank-you-linuxone to vbt
* extened strip to have .is-deep and .is-bordered for --light and --dark strips

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/download/server/thank-you-linuxone)
- compare to the [design](https://www.dropbox.com/search/work?path=%2F00_web_team%2F_ubuntu.com%2Fnew+projects%2Fvanilla+update%2Fpatterns+inventory&preview=download-server-linuxone-thank-you.png&qsid=27984850060468009768774727364561&query=linuxone) and the [demo](http://www.ubuntu.com-1645-download-thank-you-linuxone.demo.haus/download/server/thank-you-linuxone)


## Issue / Card

Fixes #1645
